docs/source/conf.py

```diff
@@ -76,6 +76,8 @@ def __getattr__(cls, name):
             return _SubDistribution
         elif "Kernel" in name or "Parallel" in name:
             return _Kernel
+        elif "__version__" in name:
+            return "1000000000.0.1"
         else:
```

Comment:
What?

Reply:
The doc build was failing because the mocked-out pytorch didn't have a `__version__` property. This fixes that :)